eng/tools/typespec-validation/src/tspconfig-rule-helper.ts

```diff
@@ -0,0 +1,154 @@
+import { Rule } from "./rule.js";
+import { RuleResult } from "./rule-result.js";
+import { TsvHost } from "./tsv-host.js";
+import { join } from "path";
+import { parse as yamlParse } from "yaml";
+
+type ExpectedValueType = string | boolean | RegExp;
```

Comment:
Are the string and boolean types ever used?  I only see RegExp passed in.

Reply:
Prepared to be used for case: `a: true`, `b: "aaa"` - strict equal